File: BE/EnglishApp/EnglishApp/Models/Classes/ClassDto.cs
using EnglishApp.Enums;
using System;

namespace EnglishApp.Models.Classes
{
    public class ClassDto
    {
        public string Name { get; set; }
        public int Id { get; set; }
        public string Description { get; set; }
        public int CourseId { get; set; }
        public string CourseName { get; set; }
        public int TeacherId { get; set; }
        public string TeacherName { get; set; }
        public string LessonName { get; set; }
        public int LessonId { get; set; }
    }
}

File: FE/englishapp/src/pages/Class/models/ClassDto.ts
interface ClassDto {
    id: number,
    name: string,
    description:string,
    courseName: string,
    courseId: number,
    teacherName: string,
    lessonName: string // Buổi học
    teacherId: number,
    lessonId: number,
    studentCount: number
}
export default ClassDto;